src/software/thunderscope/robot_communication.py

```diff
@@ -70,8 +70,13 @@ def __init__(
         self.run_world_thread = threading.Thread(target=self.run_world)
         self.run_primitive_set_thread = threading.Thread(target=self.run_primitive_set)
 
+        # initialising the estop
+        # tries to access a plugged in estop. if not found, defaults to keyboard estop
```

Comment:
I thought we only default to keyboard estop if `--use_keyboard_estop` is passed?